### ISSUE TITLE
taskcluster: sinon is not defined

### ISSUE THREAD

**Issue by mythmon** [mozilla/normandy#809]
I suspect that this is a problem where mozilla-central changed underneath us, since we use sinon from m-c.

```
0:00.81 LOG: Thread-11 ERROR Unexpected exception ReferenceError: sinon is not defined at /root/mozilla-central/obj-x86_64-pc-linux-gnu/_tests/xpcshell/browser/extensions/shield-recipe-client/test/unit/test_NormandyApi.js:96
test_getApiUrlSlashes@/root/mozilla-central/obj-x86_64-pc-linux-gnu/_tests/xpcshell/browser/extensions/shield-recipe-client/test/unit/test_NormandyApi.js:96:9
async*inner@/root/mozilla-central/obj-x86_64-pc-linux-gnu/_tests/xpcshell/browser/extensions/shield-recipe-client/test/unit/test_NormandyApi.js:21:13
async*inner@utils.js:19:35
async*asyncFunction@resource://gre/modules/Task.jsm:243:18
Task_spawn@resource://gre/modules/Task.jsm:168:12
_run_next_test@/root/mozilla-central/testing/xpcshell/head.js:1548:9
run@/root/mozilla-central/testing/xpcshell/head.js:710:9
_do_main@/root/mozilla-central/testing/xpcshell/head.js:222:5
_execute_test@/root/mozilla-central/testing/xpcshell/head.js:550:5
@-e:1:1
```

**Comment by Osmose** [comment]
Just saw this in #813, I think there's still a problem here.

I was looking into some other issues with head.js in test files yesterday, I think that might be the real culprit. I'll take this.

**Comment by mythmon** [comment]
#813 doesn't have the fix from #810. Commit a25eaf4 is still [using sinon 1.6](https://github.com/Osmose/normandy/blob/a25eaf4e93725ba0afca15f89fb9dd20a69fb04e/recipe-client-addon/test/unit/head_xpc.js#L32)
